base_import_async/__manifest__.py

```diff
@@ -11,8 +11,15 @@
     "website": "https://github.com/OCA/queue",
     "category": "Generic Modules",
     "depends": ["base_import", "queue_job"],
-    "data": ["data/queue_job_function_data.xml", "views/base_import_async.xml"],
-    "qweb": ["static/src/xml/import.xml"],
+    "data": ["data/queue_job_function_data.xml"],
+    "assets": {
+        "web.assets_backend": [
+            "base_import_async/static/src/js/import.js",
+        ],
+        'web.assets_qweb': [
+            'base_import_async/static/src/xml/import.xml',
+        ],
+    },
     "installable": False,
```

Comment:
Shouldn't the module be flagged as installable now?

Reply:
It could be, but not should. (in v_14 it was False)
This PR is about necessary changes.